drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -594,9 +632,11 @@ function DragAndDropBlock(runtime, element, configuration) {
     DragAndDropBlock.STANDARD_MODE = 'standard';
     DragAndDropBlock.ASSESSMENT_MODE = 'assessment';
 
+    var MEDIA_QUERY = '(max-width: 1001px)';
+
     var Selector = {
         popup_box: '.popup',
-        close_button: '.popup .close-feedback-popup-button'
+        close_button: window.matchMedia( MEDIA_QUERY ).matches ? '.popup .close-feedback-popup-mobile-button' : '.popup .close-feedback-popup-button'
```

Comment:
No need for media query, you can set the selector to match both buttons:

```
close_button: '.popup .close-feedback-popup-mobile-button, .popup .close-feedback-popup-button'
```

Reply:
Or maybe event better to add the same class to both. Maybe replace the current usage of `close-feedback-popup-button` with `close-feedback-popop-desktop-button`, and then add `close-feedback-popup-button` to both buttons to be able to match them with a single selector.